src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,16 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    //Drivetrain Falcon 500s
+    public static final int driveLeftLeaderFalconID = 1;
+    public static final int driveLeftFollowerFlaconID = 2;
+    public static final int driveRightLeaderFalconID = 3;
+    public static final int driveRightFollowerFalconID = 4;
+    public static final double kTurnClamp = 0;
+    public static final double kDriveMinimumSpeed = 0;
+    public static int kDriveMinimumSPeed;
```

Comment:
this looks like an extra variable? should things be using the `final double` defined on L22?

Reply:
Misspelling = extra variable. 
You are correct, I changed them to int.

Reply:
Fixed on my file.